app/core/Tracker2/DriftTracker.js

```diff
@@ -72,11 +83,40 @@ export default class DriftTracker extends BaseTracker {
     // Show when a message is received
     window.drift.on('message', (e) => {
       if (!e.data.sidebarOpen) {
-        this.driftApi.widget.show()
+        if (this.isChatEnabled) {
+          this.driftApi.widget.show()
+        }
       }
     })
   }
 
+  routeUpdated () {
+    this.updateDriftConfiguration()
+  }
+
+  userRoleUpdated () {
+    this.updateDriftConfiguration()
+  }
+
+  get onPlayPage () {
+    const { route } = this.store.state
+    return (route.path || '').indexOf('/play/') === 0
```

Comment:
Should we match the start of the path here, just in case we end up with future URLs like `/benefits/play/` or `/teachers/guides/play/python/` and run into a surprise with Drift chats disappearing?

Reply:
I may be wrong, but by checking that the index is `0` should only pass this check if `/play/` is at the start.

Nitpick: this could maybe be made more explicit by using [`String.prototype.startsWith`](https://caniuse.com/#search=startswith), but indexOf has best support.